cloud/transport/cluster.go

```diff
@@ -107,7 +107,7 @@ func (c *Cluster) serve() error {
 	c.grpcServer = grpc.NewServer()
 	cmd.RegisterClusterServiceServer(c.grpcServer, c)
 	if err := c.grpcServer.Serve(c.ln); err != nil {
-		c.log.Error("serving incoming requests", "error", err)
+		c.log.Error("serving incoming requests: " + err.Error())
```

Comment:
Side suggestion: would it make sense here to follow a more "structured" log approach where we would have the error in a dedicated json field ?

Reply:
As the logging is already structured, this is simply about enhancing the error context

Reply:
As the logging is already structured, this is simply about enhancing the error context